src/common/constants.ts

```diff
@@ -141,7 +141,7 @@ export const setColumnToBackendTerms: any = (listName: string, columnName: strin
   };
 
   const additionalItems: any = {
-    flow: { TITLE: 'name' },
+    flow: { TITLE: 'name', ' ': 'is_pinned' },
```

Comment:
This does not feel right somehow. We should think about fixing this sometime soon. Creating an issue for this https://github.com/glific/glific-frontend/issues/2097

Reply:
Yes, it is weird :) We map using what's displayed in the UI which is not the best way. This is also an issue when it comes to localising it.  So this is only a workaround. I will take a look at clean-up once done with package stuff.